frontend/src/environments/environment.prod.ts

```diff
@@ -11,4 +11,7 @@ export const environment = {
         appId: '1:681019224003:web:f114bfb64139e3146a1776',
         measurementId: 'G-CZQ7Z16TJZ',
     },
+    zoom: {
+        clientId: 'f6QaAcMFT62EN0FdLBxCTQ',
```

Comment:
Is this the same client Id as we have in the backend?

Reply:
Yep

Reply:
it would be better to save it in one place

Reply:
It's saved in environment variable, should we access it on frontend somehow from there?